app/repositories/repository_registry.rb

```diff
@@ -27,7 +28,15 @@ def scoped_manual_repository(collection)
         SectionAssociationMarshaller.new(
           section_repository_factory: section_repository_factory,
           decorator: ->(manual, attrs) {
-            DocumentFactoryRegistry.new.manual_with_sections.call(manual, attrs)
+            ManualValidator.new(
```

Comment:
I probably could, but my goal was simply to remove one of the last classes/modules including the word "Document" and these changes achieve that. I think I'd prefer to limit the scope of this PR to just that.